front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/V2PipelineTemplateController.java

```diff
@@ -145,10 +172,34 @@ private void checkForDuplicatePipelineTemplate(String id) {
     throw new DuplicateEntityException("A pipeline template with the id " + id + " already exists");
   }
 
+  private String computeSHA256Digest(PipelineTemplate pipelineTemplate) {
```

Comment:
I might throw some unit tests in here to verify that this correctly computes the same hash for two pipelines with different ordered keys.